src/generated/api/collectionsApi.ts

```diff
@@ -539,12 +566,16 @@ export class CollectionsApi {
   /**
    * Retrieve a list of collections in an account.
    * @summary List collections
+   * @param accountId The account that owns this set of collections, e.g. &#x60;1618535966441231024&#x60;.
    * @param pageSize The maximum number of collections to return. The service may return fewer than this value.  If unspecified, at most 50 collections are returned.  The maximum value is 100; values above 100 are coerced to 100.
    * @param pageToken A page token, received from a previous [ListCollections](/docs/api#operation/ListCollections) call.  Provide this to retrieve the subsequent page.  When paginating, all other parameters provided to [ListCollections](/docs/api#operation/ListCollections) must match the call that provided the page token.
+   * @param view The amount of information to include in each retrieved collection.   - BASIC: Include basic information including display name and domains. This is the default value (for both [ListCollections](/docs/api#operation/ListCollections) and [GetCollection](/docs/api#operation/GetCollection)).  - FULL: Include the information from &#x60;BASIC&#x60;, plus full collection details like disk usage.
    */
   public async listCollections(
+    accountId?: string,
```

Comment:
@jonathaningram would this be a breaking change?

Reply:
Yep it would be.